cacerts/detect.go

```diff
@@ -0,0 +1,65 @@
+package cacerts
+
+import (
+	"path/filepath"
+	"sort"
+	"strings"
+
+	"github.com/buildpacks/libcnb"
+	"github.com/paketo-buildpacks/libpak/bard"
+)
+
+type Detect struct {
+	Logger bard.Logger
+}
+
+const (
+	BindingType = "ca-certificates"
+)
+
+// Detect always passes and optionally provides ca-certificates. If there is a binding of
+// type "ca-certficates" Detect also requires ca-certificates and provides an array of cerficate paths in the
+// plan entry metadata.
+func (d Detect) Detect(context libcnb.DetectContext) (libcnb.DetectResult, error) {
+	paths := getsCertsFromBindings(context.Platform.Bindings)
+	if len(paths) > 0 {
+		return libcnb.DetectResult{Pass: true, Plans: []libcnb.BuildPlan{{
+			Provides: []libcnb.BuildPlanProvide{
+				{Name: "ca-certificates"},
+			},
+			Requires: []libcnb.BuildPlanRequire{
+				{
+					Name: "ca-certificates",
+					Metadata: map[string]interface{}{
+						"paths": paths,
+					},
+				},
+			},
+		}}}, nil
+	}
+
+	return libcnb.DetectResult{
+		Pass: true,
+		Plans: []libcnb.BuildPlan{
+			{
+				Provides: []libcnb.BuildPlanProvide{
+					{Name: "ca-certificates"},
+				},
+			},
+			{}, // always contribute runtime helper
```

Comment:
This isn't required in order to always contribute a runtime helper.

Reply:
I believe it is currently b/c this the previous plan in this result has a provides with no matching requires. However it wouldn't be necessary if I incorporated your suggestion above -> https://github.com/paketo-buildpacks/ca-certificates/pull/1#discussion_r515427550

Reply:
Resolved by explicitly adding a `ca-cert-helper` entry.